src/resources/assets/js/Checkout/index.js

```diff
@@ -3,6 +3,10 @@ import MxCartDiscount from './Components/CartDiscount.vue'
 import MxStripe from './Components/Stripe.vue'
 import MxAddress from './Components/AddressLookup.vue'
 import MxDelivery from './Components/DeliverySelector.vue'
+import DeliveryMonth from './Components/Delivery/DeliveryMonth.vue'
+import DeliveryWeek from './Components/Delivery/DeliveryWeek.vue'
```

Comment:
```suggestion
import DeliveryWeek from './Components/Delivery/DeliveryWeek.vue'
```

If it is only used inside another vue component this could be imported locally. TBH, not entirely sure if there is any benefit ... just saying. ;)

Reply:
Ah yea fair play, I'll import it locally